pkg/cmd/kind/load/docker-image/docker-image.go

```diff
@@ -120,7 +120,7 @@ func runE(logger log.Logger, flags *flagpole, args []string) error {
 		id, err := nodeutils.ImageID(node, imageName)
 		if err != nil || id != imageID {
 			selectedNodes = append(selectedNodes, node)
-			logger.V(0).Infof("Image: %q with ID %q not present on node %q", imageName, imageID, node.String())
+			logger.V(0).Infof("Image: %q with ID %q not yet present on node %q, loading...", imageName, imageID, node.String())
```

Comment:
`loading ...` implies we're doing this right now which is somewhat misleading, we won't actually load it.

This should probably be V(1) / debug anyhow, that got lost in translation at some point refactoring the logging https://github.com/kubernetes-sigs/kind/blame/5df15c23d3a8520193655460df310540c2f1a7d3/cmd/kind/load/docker-image/docker-image.go#L122